### PR TITLE
mac-capture: Log CoreAudio device sample rate

### PR DESCRIPTION
**Pull request by gxalpha**
### Description

Adds the sample rate of an audio input/output capture device to the log file.
image

### Motivation and Context

Requested for support purposes.
Useful to easily see sample rate mismatches.

### How Has This Been Tested?

macOS 13 Beta 10
Initialized different devices, saw the sample rate being logged (see above).

### Types of changes

- Tweak (non-breaking change to improve existing functionality)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: plugins/mac-capture/mac-audio.c [hunk 1]
```diff
@@ -629,7 +629,8 @@ static bool coreaudio_init(struct coreaudio_data *ca)
 	if (!coreaudio_start(ca))
 		goto fail;
 
-	blog(LOG_INFO, "coreaudio: device '%s' initialized", ca->device_name);
+	blog(LOG_INFO, "coreaudio: Device '%s' [%d Hz] initialized",
+	     ca->device_name, ca->sample_rate);
```
Comment (jp9000) [inline plugins/mac-capture/mac-audio.c:633]: Include <inttypes.h>, then change string to:
`"coreaudio: Device '%s' [%" PRIu32 " Hz] initialized"`
instead

Comment (PatTheMav) [reply plugins/mac-capture/mac-audio.c:633]: `PRIu32` resolves to `u` on macOS - given that mac-capture is macOS only, I don't think we necessarily need a portable macro?

Comment (gxalpha) [reply plugins/mac-capture/mac-audio.c:633]: Fixed

Comment (jp9000) [reply plugins/mac-capture/mac-audio.c:633]: Sorry about that, already changed though so either way is fine I suppose. I'll mark as resolved for now

**Pull request merged**